paper-tabs.html

```diff
@@ -160,28 +159,28 @@
 
   <template>
 
-    <div class-name="[[scrollButtonClass]]">
-      <core-icon icon="paper-tabs:chevron-left" class-name="[[leftIconClass]]"
+    <div class$="[[computeScrollButtonClass(scrollable, hideScrollButton)]]">
```

Comment:
Yup, they're named [annotated computed properties](https://github.com/Polymer/polymer/blob/0.8-preview/PRIMER.md#annotated-computed) in the primer doc.